packages/client/src/generation/generateClient.ts

```diff
@@ -135,6 +135,16 @@ export async function buildClient({
     fileMap['edge.d.ts'] = await TS(edgeTsClient, true)
   }
 
+  if (generator?.previewFeatures.includes('denoDeploy')) {
+    // We use modified version of edge.js in denoDeploy mode.
+    fileMap['edge.js'] = await JS(edgeTsClient, true)
+    fileMap['edge.ts'] = `import 'https://deno.land/std@0.152.0/dotenv/load.ts'
```

Comment:
Assuming that we don't add a Deno folder and keep it this way. How would we name the entry point for the regular Deno runtime? Would it be `index.ts`?

Reply:
Oops, I forgot we talked about `/deno/` directory. I'll move deno specific files there.